public/js/core.js

```diff
@@ -59,8 +59,13 @@ tarchive.controller('CoreController', ['$scope', '$filter', 'API', 'NgTableParam
 
 	$scope.askForKey = function() {
 		swal({
-			title: "Enter your key!",
-			text: "Enter the key provided by the bot:",
+			html: true,
+			title:	'Welcome to Tarchivepanel!',
+			text:		'What is Tarchivebot? <br><br> A Telegram bot that archives your chat and provides you with a web interface to look up messages and statistics. <br><br> Add the bot using this link: ' +
+							'<a href="https://t.me/Tarchivebot" target="_blank">Tarchivebot</a><br>' +
+							'The bot requires read access to messages! <br><br>' +
+							'To retrieve your key simply type /apikey into chat<br><br>' +
```

Comment:
Maybe try putting`/apikey` into a `<pre>` tag if it looks good. Not sure if it will in practice though.